packages/react/src/provider/use-context-mutator.ts

```diff
@@ -0,0 +1,28 @@
+import { useCallback, useContext, useRef } from 'react';
+import { OpenFeature, EvaluationContext } from '@openfeature/web-sdk';
+import { Context } from './context';
+
+/**
+ *
+ * A hook for accessing context mutating functions.
+ *
+ */
+export function useContextMutator() {
```

Comment:
I wonder if we should add an optional param here for using the scoped OR global evaluation context.

By default, we could use the context indicated by the associated `domain`, but if desired, the user could indicate they want to mutate the global evaluation context.

Reply:
Yes, I think that's a good idea, given how context works with domains.

@wichopy, [here's](https://openfeature.dev/docs/reference/technologies/client/web/#manage-evaluation-context-for-domains) a quick overview of how context works with domains.

Reply:
done!